src/main/java/me/xmrvizzy/skyblocker/config/SkyblockerConfig.java

```diff
@@ -67,7 +67,7 @@ public static class QuickNav {
 
         @ConfigEntry.Category("button3")
         @ConfigEntry.Gui.CollapsibleObject()
-        public QuickNavItem button3 = new QuickNavItem(true, new ItemData("bone"), "\\Pets \\(\\d+/\\d+\\)", "/pets");
+        public QuickNavItem button3 = new QuickNavItem(true, new ItemData("bone"), "Pets(:? \\(\\d+\\/\\d+\\))?", "/pets");
```

Comment:
Still the same look what I mean:

- with your change `Pets(:? \\(\\d+\\/\\d+\\))?`
![grafik](https://github.com/SkyblockerMod/Skyblocker/assets/19829407/8f5d49c5-77ec-4214-935c-d84b5b0972b4)

- with my change `Pets( \(\d+\/\d+\))?`
![grafik](https://github.com/SkyblockerMod/Skyblocker/assets/19829407/d765c995-607c-4c06-9908-37e498828104)

Reply:
i have tested it and the latest one work for me in all conditions, no matter if i have only one pet page or more
![image](https://github.com/SkyblockerMod/Skyblocker/assets/49841713/e9d5a56e-2a34-4c39-bf46-1893a03e5422)
![image](https://github.com/SkyblockerMod/Skyblocker/assets/49841713/151a3c15-53e0-48af-93db-0da08f86402c)
![image](https://github.com/SkyblockerMod/Skyblocker/assets/49841713/7c6cfccb-d89f-46be-85dd-df9f958aff69)